src/main/webapp/app/index.scss

```diff
@@ -110,7 +110,8 @@ b {
   }
 }
 
-.news {
+.news,
+.yearEndReview {
```

Comment:
I think we use `year-end-review` format?

Reply:
Yes in the scss file, but if it's in the modules.scss file, we do camel case. Let me fix.